sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/MinedOpinion.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.util.IterableStream;
+
+/**
+ * The {@link MinedOpinion} model.
+ */
+public final class MinedOpinion {
+    private final AspectSentiment aspect;
+    private final IterableStream<OpinionSentiment> opinions;
+
+    /**
+     * Create an {@link MinedOpinion} model that describes mined opinion.
+     *
+     * @param aspect The aspect of a product/service that this opinion is about.
+     * @param opinions The opinions of the aspect text.
+     */
+    public MinedOpinion(AspectSentiment aspect, IterableStream<OpinionSentiment> opinions) {
+        this.aspect = aspect;
+        this.opinions = opinions;
+    }
+
+    /**
+     * Get the opinions of aspect text.
```

Comment:
Should add more docs explaining what aspect and opionion is.

Reply:
updated